source/docs/software/wpilib-tools/robot-characterization/generating-project.rst

```diff
@@ -0,0 +1,62 @@
+Generating a Project
+====================
+
+To use the toolsuite, we first need to generate a robot project.
+
+Select Project Location
+-----------------------
+
+First, select the desired project location on the new project GUI:
+
+.. image:: images/selecting-project-location.png
+   :alt: Selecting the project location in the robot characterization GUI
+
+Select Project Type
+-------------------
+
+Next, select your project type from the drop-down menu:
+
+.. image:: images/selecting-project-type.png
+   :alt: Selecting the project type in the robot characterization GUI
+
+
+- **Simple** projects use encoders plugged into the RIO's DIO ports, and measure voltage with the PDP.
+- **Talon** projects use encoders plugged into a Talon SRX, and measure voltage with the Talon.
+- More project types may be added in the future.
+
+Configure Project Parameters
+----------------------------
+
+In order to run on your robot, the tool must know some parameters about how your robot is set up. Project config settings are formatted as a `Python dictionary literal <https://www.linuxtopia.org/online_books/programming_books/python_programming/python_ch15s02.html>`__. These can be modified via the in-window config editor:
+
+.. image:: images/config-editor.png
+   :alt: Using the robot characterization configuration editor
+
+Take care of the following caveats when entering your robot specifications:
+
+- The key names must *not be changed*, as they are hard-coded for each project type. Only the values (i.e. the things on the right-hand side of the colons) should be modified.
+- ``True`` and ``False`` *must* be capitalized, as they are evaluated as native Python.
+- All string values (e.g. controller names and unit types) *must* be wrapped in quotes and *must* correspond exactly to one of the options described.
+- All list values *must* be lists, even if they have only one element. Do not omit trailing commas (e.g. in ``(False,)``), as this will cause them not to be parsed as a list.
```

Comment:
These are actually tuples.
I'm not sure if using a list would work (probably would, as tuples are just immutable lists and python is flexible about types)

Reply:
@Oblarg How should this be revised?

Reply:
By replacing the parens with square brackets.  I'll do that now.